RoboJackets/robocup-software
# Fix settle's end condition

## Description
Originally, this PR was supposed to be about creating an intercept skill to make up for the lack of an intercept planner. However, along the way I realized that settle.is_done() always returned true due to some typing errors in create_robot() in world_state_converter.py. After fixing this and tuning the min speed param, settle seems to work pretty well for getting in front of the ball, could probably use some tuning at the planning level though. I left the intercept skill there, it just moves a robot to the closest point on the straight line trajectory of the ball.

## Steps to test
### Testing Settle
1. Open gameplay_node, and set the test_play_selector to use some passing play
2. Open sim and let the robots play
3. Observe Passing

Expected result: The receiving robot should get in front of the ball and try to face it.

### Testing Intercept
1. Create a play that uses the intercept skill (preferably defense)
2. Open sim2play and run the defense play against a passing play
3. Observe

Expected Result: The intercepting robot should try to get along the path of the robot
